content/en/docs/setup/install/helm/index.md

```diff
@@ -17,14 +17,29 @@ Follow this guide to install and configure an Istio mesh using
 
 ## Installation steps
 
-1. Create a namespace `istio-system` for Istio components:
+This section describes the procedure to deploy Istio Service Mesh using Helm. The general syntax for helm installation is:
+`helm install <release> <chart> --namespace <namespace> --create-namespace [--set <other_parameters>]`
+
+The variables specified in the command are as follows:
+
+    <chart> : A path to a packaged Helm chart, unpackaged Helm chart or a URL.
+    <release> : Name to identify and manage the Helm chart once installed.
+    <namespace> : Namespace in which the chart has to be installed.
```

Comment:
```suggestion
This section describes the procedure to install Istio using Helm. The general syntax for helm installation is:

{{< text syntax=bash snip_id=none >}}
$ helm install <release> <chart> --namespace <namespace> --create-namespace [--set <other_parameters>]
{{< /text >}}

The variables specified in the command are as follows:

* `<chart>` A path to a packaged Helm chart, unpackaged Helm chart, or a URL.
* `<release>` A name to identify and manage the Helm chart once installed.
* `<namespace>` The namespace in which the chart is to be installed.

Reply:
I don't know much about helm, but this seems unclear:

`A path to a packaged Helm chart, unpackaged Helm chart, or a URL.`

What's the URL to? Would this be correct:

`A file path or URL to a packaged or unpackaged Helm chart.`